crates/cast/bin/args.rs

```diff
@@ -898,6 +898,14 @@ pub enum CastSubcommand {
         command: WalletSubcommands,
     },
 
+    /// Download a contract creation code from Etherscan and RPC.
+    #[command(visible_alias = "cc")]
+    CreationCode(CreationCodeArgs),
+
+    /// Display args used for the contract initialization
+    #[command(visible_alias = "cra")]
+    CreationArgs(CreationArgsArgs),
```

Comment:
Does this always refer to constructor arguments?

If so, can we rename it `ConstructorArgs`?